tasks/files/snapshot.py

```diff
@@ -1171,14 +1061,16 @@ def umount_snapshot_set(snapset_json, verify_only, check_mode):
 
         if "mount_origin" in list_item:
             origin = bool(list_item["mount_origin"])
```

Comment:
I know it isn't part of this commit, but what is the `bool` being used for?  If it is possible that `list_item["mount_origin"]` is not of type `bool` e.g. a string like `"True"` or `"False"`, and you want to convert that to a `bool`, then you'll need to use something else - https://stackoverflow.com/questions/715417/converting-from-a-string-to-boolean-in-python

Reply:
I'll replace with:

def to_bool(bool_string):
    return bool_string.lower() in ['true', '1', 't', 'y']

Reply:
nah - the Ansible module code has built-in argument validation - it can wait